pkg/controller/operators/catalog/operator.go

```diff
@@ -255,6 +258,24 @@ func NewOperator(ctx context.Context, kubeconfigPath string, clock utilclock.Clo
 		return nil, err
 	}
 
+	operatorGroupInformer := crInformerFactory.Operators().V1().OperatorGroups()
```

Comment:
should we add a comment about why this is needed? (non-blocking)

Reply:
I think we're okay, its a pretty common standard to define informers when you want to list objects in a controller.